bin/hdfcoinc/pycbc_page_injtable

```diff
@@ -26,15 +28,41 @@ else:
     title = "Found Injections"
     found = f['found_after_vetoes']
     idx = found['injection_index'][:]
-    tdiff = (found['time1'][:] - found['time2'][:]) * 1000
-    tdiff_str = '%s - %s time (ms)' % (f.attrs['detector_1'], f.attrs['detector_2'])
+    if('detector_1' in f.attrs):
+        tdiff = (found['time1'][:] - found['time2'][:]) * 1000
+        tdiff_str = '%s - %s time (ms)' % (f.attrs['detector_1'], f.attrs['detector_2'])
+        ids = {f.attrs['detector_1']: found['trigger_id1'][:],
+               f.attrs['detector_2']: found['trigger_id2'][:],}
+        found_cols = [tdiff, found['stat'], found['ifar'], found['ifar_exc']]
+        found_names = [tdiff_str, 'Ranking Stat.', 'Inc. IFAR (yrs)', 'Exc. IFAR']
+        found_formats =  ['##.##', '##.##', '##', '##']
+
+    else:
+        detectors = ['L1','H1','V1','K1','I1']
```

Comment:
You should be able to use something like `f.attrs['ifos'].split(' ')` to get a list of the detectors used in this file